easybuild/easyconfigs/i/icc/README.md

```diff
@@ -0,0 +1,13 @@
+In order to use these configs you must first download the Intel Parallel Studio XE CPP file. NOT the full bundle! The file should look like this: parallel_studio_xe_2018_update2_composer_edition_for_cpp.tgz    
+
+Next you must specify the license file. There are three options:  
+* Place it in the following path: ~/licenses/intel/license.lic
+* Set an environment variable: `export INTEL_LICENSE_FILE=/path/to/file.lic`
+* Set an environment variable: `export LM_LICENSE_FILE=/path/to/file.lic`
```

Comment:
There's a 4th option: change the `license_file` spec in the easyconfig file you are using

Reply:
Again, good to know but not sure that is what is wanted in a "heres how you get this to work" document for those getting started.

Reply:
Again, just for completeness sake, it should be mentioned...

Reply:
Done. Just added to both.